core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -55,51 +55,90 @@
+             (mouseover)="openSubmenu($event, 'learnMenu')"
              (mouseleave)="closeSubmenuIfNotMobile($event)"
              autofocus>
-            <span class="oppia-navbar-tab-content"
-                  [innerHTML]="'I18N_TOPNAV_CLASSROOM' | translate">
+            <span class="oppia-navbar-tab-content">
+              {{ 'I18N_TOPNAV_LEARN' | translate }}
             </span>
           </a>
           <ul ngbDropdownMenu
-              class="dropdown-menu dropdown-menu oppia-navbar-dropdown"
-              (mouseover)="openSubmenu($event, 'classroomMenu')"
+              [ngStyle]="{'left.px': learnDropdownOffset}"
+              [ngClass]="{'classroom-disabled': !CLASSROOM_PROMOS_ARE_ENABLED, 'classroom-enabled': CLASSROOM_PROMOS_ARE_ENABLED}"
+              class="dropdown-menu oppia-navbar-dropdown learn-dropdown-menu"
+              (mouseover)="openSubmenu($event, 'learnMenu')"
               (mouseleave)="closeSubmenuIfNotMobile($event)"
               role="menu"
               autofocus>
             <li class="nav-item">
-              <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})"
-                 (click)="navigateToClassroomPage('/learn/math')"
-                 class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link"
-                 href="#"
-                 [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate">
-              </a>
+              <div class="nav-item-main d-flex">
+                <div *ngIf="CLASSROOM_PROMOS_ARE_ENABLED" class="nav-item-left d-flex flex-column justify-content-between">
+                  <div class="nav-item-top d-flex justify-content-between">
+                    <div class="nav-content d-flex flex-column">
+                      <a (keydown)="onMenuKeypress($event, 'learnMenu', {shiftTab: ACTION_CLOSE})"
+                         (click)="navigateToClassroomPage('/learn/math')"
+                         class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link nav-item-top-link"
+                         href="#">
+                          {{ 'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate }}
+                      </a>
+                      <div class="des">{{ 'I18N_TOPNAV_LEARN_DESCRIPTION_1' | translate }}</div>
+                    </div>
+                    <picture>
+                      <source type="image/png"
+                              [srcset]="getStaticImageUrl('/topnav/learn.png')">
+                      <source type="image/webp"
+                              [srcset]="getStaticImageUrl('/ltopnav/learn.webp')">
+                      <img [src]="getStaticImageUrl('/topnav/learn.png')"
+                           class="learn"
+                           alt="Learn">
+                    </picture>
+                  </div>
+                  <div class="nav-item-bottom d-flex flex-column" *ngIf="!userIsLoggedIn">
```

Comment:
Hi @Lawful2002, why do we surface the topics only if the user is logged out? This seems unexpected and had to be reverted in https://github.com/oppia/oppia/pull/15125.

Reply:
The [mocks](https://www.figma.com/file/6XBtQqou0AHS2WWg345dmu/Nav-%26-Library?node-id=737%3A648) for the top-nav bar suggested that the 'logged-in' state should contain the 'Continue Learning' link instead of the topic list.